packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -75,6 +75,7 @@ const shouldHandleIndentationEvent = (indenting: boolean) => (
 ) => {
     const { keyCode, altKey, shiftKey, ctrlKey, metaKey } = event.rawEvent;
     return (
+        !Browser.isMac &&
```

Comment:
Maybe we can add this check to the defaultDisabled parameter of the ContentEdit Feature instead, Like in `OutdentWhenEnterOnEmptyLine`